src/super_gradients/wandb/artifacts.py

```diff
@@ -0,0 +1,13 @@
+import os
+import wandb
+
+
+def save_artifact(path):
```

Comment:
If you define the type to be model, please name the function `save_wandb_model_artifact`
otherwise, call it `save_wandb_artifact(path: str, type: str)`.

note the added explicit typing

Reply:
please add documentation:
something like:
```
 upload and wandb Artifact.
Note that this function can be called only after wandb.init()
 :param path: the local full path to the pth file to be uploaded 
 ```

Reply:
looks like this code is used only inside the `WandBSGLogger`
why not include this as a private function of `WandBSGLogger`?